jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/tags/users.tag

```diff
@@ -25,18 +25,6 @@
   <c:when test="${!(empty users)}">
     <c:forEach var="user" items="${users}" varStatus="i">
       <c:set var='labelClass' value=''/>
-      <c:if test="${branch != null}">
```

Comment:
Don't forget to write a comment in the task that this functionality was removed.